atoma-confidential/src/service.rs

```diff
@@ -252,7 +252,7 @@ impl AtomaConfidentialCompute {
                 service_shared_secret_receiver,
                 shutdown_signal,
             )?;
-            if service.is_cc_supported {
+            if service.is_cc_supported || service.is_ppcie_enabled {
```

Comment:
if cc is supported wouldn't it imply ppcie is enabled? or are we providing cc inference without ppcie?